.gitignore

```diff
@@ -29,3 +29,4 @@ client/src/upload/
 # ignore configuration
 .env.production
 .env.development
+integrations.log
```

Comment:
This was probably generated by your LOG_FILE variable in `.env.development` or `.env.production`.  Check to see if that is the case.